src/main/java/com/sucy/skill/dynamic/trigger/ChatTrigger.java

```diff
@@ -0,0 +1,66 @@
+package com.sucy.skill.dynamic.trigger;
+
+import com.sucy.skill.api.Settings;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.player.AsyncPlayerChatEvent;
+
+import java.util.Map;
+
+/**
+ * ProSkillAPI © 2023
+ * com.sucy.skill.dynamic.trigger.BlockBreakTrigger
+ */
+public class ChatTrigger implements Trigger<AsyncPlayerChatEvent> {
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getKey() {
+        return "CHAT";
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Class<AsyncPlayerChatEvent> getEvent() {
+        return AsyncPlayerChatEvent.class;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean shouldTrigger(final AsyncPlayerChatEvent event, final int level, final Settings settings) {
+        boolean     cancelMessage   = settings.getBool("cancel",false);
+        String      message         = settings.getString("message", "");
```

Comment:
We should also add an option here to match with regex instead of just a `contains`